libbsd: 0.11.3 -> 0.11.5

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/libbsd/versions.


meta.description for libbsd is: "Common functions found on BSD systems"

meta.homepage for libbsd is: "https://libbsd.freedesktop.org/"

meta.changelog for libbsd is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 0.11.5 with grep in /nix/store/nffc0fqd9n8wvavibp8rnvf5h9wadi24-libbsd-0.11.5
- found 0.11.5 in filename of file in /nix/store/nffc0fqd9n8wvavibp8rnvf5h9wadi24-libbsd-0.11.5

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
485 total rebuild path(s)

432 package rebuild(s)

432 x86_64-linux rebuild(s)
53 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
MIDIVisualizer
acme-client
adapta-gtk-theme
almanah
arcan.all-wrapped
arcan.arcan
arcan.arcan-wrapped
arcan.durden-wrapped
arcan.ffmpeg
arcan.pipeworld-wrapped
arcan.prio-wrapped
arcan.xarcan
aucatctl
autofs5
bareos
bitwig-studio1
bitwig-studio2
bluez-alsa
bottles
bubblemail
bumblebee
calls
ceph
ceph-client
ceph-csi
ceph-dev
chatty
chrome-gnome-shell
cinnamon.cinnamon-common
cinnamon.cinnamon-control-center
cinnamon.cinnamon-gsettings-overrides
cinnamon.cinnamon-screensaver
cinnamon.nemo
clementine
clementineUnfree
collectd
corrscope
csmith
diffoscope
docker-machine-kvm
docker-machine-kvm2
dpdk
dropbox-cli
easysnap
eid-mw
elementary-planner
empathy
enum4linux
enum4linux-ng
evolution
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/nffc0fqd9n8wvavibp8rnvf5h9wadi24-libbsd-0.11.5 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A libbsd https://github.com/r-ryantm/nixpkgs/archive/373211d6a937e4f90fa237a471fecf00ef4f8e6c.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/nffc0fqd9n8wvavibp8rnvf5h9wadi24-libbsd-0.11.5
ls -la /nix/store/nffc0fqd9n8wvavibp8rnvf5h9wadi24-libbsd-0.11.5/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @matthewbauer for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
libbsd: 0.11.3 -> 0.11.5

c0bw3b reviewed (commented):
I've rebuilt a good chunk of its reverse-deps:

Result of `nixpkgs-review pr 159492` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>aucatctl</li>
    <li>csmith</li>
    <li>eid-mw</li>
    <li>fwts</li>
    <li>health-check</li>
    <li>hfsprogs</li>
    <li>libbsd</li>
    <li>opendkim</li>
    <li>samba</li>
    <li>signify</li>
    <li>stress-ng</li>
    <li>ustreamer</li>
  </ul>
</details>

`aucatctl` runs ok

c0bw3b committed:
libbsd: clarify licensing

Upstream references:
https://gitlab.freedesktop.org/libbsd/libbsd/-/blob/0.11.5/COPYING
